apps/publications/src/assets/i18n/en.json

```diff
@@ -1,3 +1,286 @@
 {
-
+  "MENU_ITEMS": {
+    "ALL_PUBLICATIONS": "All publications",
+    "MY_PUBLICATIONS": "My publications",
+    "CREATE_PUBLICATION": "Create publications",
+    "AUTHORS": "Authors",
+    "CATEGORIES": "Categories",
+    "PUBLICATION_SYSTEMS": "Publication systems"
+  },
+  "HEADER": {
+    "TITLE": "Publications",
+    "LOGOUT": "Log out"
+  },
+  "CATEGORIES_LIST": {
```

Comment:
the file en.json for publications needs to be that long?
also you are copying some of the labels, arent you? Thats always not a good way of doing things.